### PR TITLE
[CAM-11106] - Review documentation explaining CAM scopes hierarchy an…

### PR DESCRIPTION
**Pull request by julio-castanar**
@gsanchezu @mirasierra 
Could you review this pull request, please?

### CODE REVIEW

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md [hunk 1]
```diff
@@ -19,8 +19,8 @@
 * [Organizations](#organizations)
 * [Switching and Managing Workspaces, Cost Centers and Organizations](#switching-and-managing-workspaces-cost-centers-and-organizations)
 * [Sharing Boxes, Instances, and Providers](#sharing-boxes-instances-and-providers)
-* [Sharing with people or groups across Organizations](#sharing-with-people-or-groups-across-organizations)
-* [Getting General Support](#getting-general-support)
+* [Federated Organizations](#federated-organizations)
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: Please, leave it as it was

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md [hunk 2]
```diff
@@ -80,26 +97,38 @@ Users who manage a big team or several project are usually Cost Center administr
 
 ### Organizations
 
+An Organization is the highest scope level used to manage several Cost Centers and the Workspaces they include.
+
+#### Context elements
+
+An Organization groups a set of Cost Centers.
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]:
```suggestion
An Organization includes a set of Cost Centers. A default Cost Center is created whenever a new Organization is created in Cloud Application Manager. Organization administrators can create as many Cost Center as they want to align with their organization structure.
```

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md [hunk 3]
```diff
@@ -134,7 +163,8 @@ To discontinue sharing with a user or workspace, open the sharing dialog, and re
 
 #### Transfer Ownership
 
-Sometimes, because your role in the organization changes, you may want to transfer an asset you own to another user or workspace. To change owners, open the sharing dialog, and make another user or workspace the owner. An asset can only have one owner at a time.
+Sometimes, because your role in the organization changes, you may want to transfer an asset you own to another user or workspace. To change owners, open the sharing dialog, and make another user or workspace the owner.  
+An asset can only have one owner at a time.
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: Either add another line to separate paragraph, or leave it as it was

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md [hunk 4]
```diff
@@ -159,9 +189,12 @@ Also you may give view access to make an instance available for others to use, s
 
 #### Sharing Providers
 
-Sharing providers has its benefits. You can give view access to company-approved providers and let users deploy to that particular provider. When teams deploy to a shared provider, you can track org-wide usage and compliance cohesively. Provider accounts can be shared only in the Enterprise Edition.
+Sharing providers has its benefits. You can give view access to company-approved providers and let users deploy to that particular provider. When teams deploy to a shared provider, you can track org-wide usage and compliance cohesively.  
+Provider accounts can be shared only in the Enterprise Edition.
 
-### Sharing with people or groups across Organizations
+### Federated Organizations
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: Leave it as it was

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md [hunk 5]
```diff
@@ -189,21 +222,23 @@ In order to share a team workspace across organizations you have to follow the n
...
 
   ![User can see the team workspace](../../images/core-concepts/user-can-see-team-workspace.png)
 
-#### A particular user stop sharing team workspaces
+#### Stop sharing in a Federated Organization
 
 To stop sharing a team workspace with other Organizations, you should be an administrator of the organization, administrator of cost center where the team workspace belongs to, or you should be the administrator of the team workspaces. In this article, we consider that the user is an organization administrator.
 
-In order stop sharing a team workspace across organizations you have to follow the next steps:
+#### Remove a particular user from a Federated Organization
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: Remove this title

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md [hunk 6]
```diff
@@ -219,9 +254,7 @@ In order stop sharing a team workspace across organizations you have to follow t
 
   ![The user don't see the team workspaces](../../images/core-concepts/user-not-see-team-workspace.png)
 
-#### Stop sharing team workspaces
-
-To revoke access of sharing a team workspace with other Organizations, you should be an administrator of the organization, administrator of cost center where the team workspace belongs to, or you should be the administrator of team workspaces. In this article, we consider that the user is an organization administrator.
+#### Stop sharing team workspaces with a Federated Organization
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: Leave this as it was

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md [hunk 7]
```diff
@@ -235,22 +268,12 @@ Click **Revoke Access** in the confirmation dialog.
...
-Customers can contact the CenturyLink Global Operations Support center (support desk) directly for getting help with Cloud Application Manager as well as any other supported product that they’ve subscribed to.  Below are three ways to get help.
-
-#### Contact:
-
-1. **Phone:** 888-638-6771
-
-2. **Email:** incident@centurylink.com
-
-3. **Create Ticket in Cloud Application Manager:** Directly within the platform, users can “Create Ticket” by clicking on the “?” symbol in upper right corner near the users log-in profile icon.  This takes users directly to the Managed Servicers Portal where they can open, track and review status of issues that have been raised with the support desk.  Additionally, this is how a TAM can be engaged as well.
+### Contacting Cloud Application Manager Support
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: Please, also leave this as it was

File: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md (hunk 4 above)
Comment (mirasierra) [reply Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md]: In CAM-11115 suggest Sharing with people or groups across Organizations
image

**Pull request merged**